install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -24,6 +24,10 @@ initContainers:
   - "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/includeInboundPorts` (includeInboundPorts .Spec.Containers) }}"
   - "-d"
   - "{{ excludeInboundPort (annotation .ObjectMeta `status.sidecar.istio.io/port` .Values.global.proxy.statusPort) (annotation .ObjectMeta `traffic.sidecar.istio.io/excludeInboundPorts` .Values.global.proxy.excludeInboundPorts) }}"
+  {{ if or (isset .ObjectMeta.Annotations `traffic.sidecar.istio.io/excludeOutboundPorts`) (ne .Values.global.proxy.excludeOutboundPorts "") -}}
```

Comment:
Not sure if we are allowing updates to these helm charts or requiring helm changes to go into the istio/installer.

Reply:
We are discussing making istio/installer an alpha feature and shipping this one as the default in the TOC today, so we will need this.

Reply:
Will need to go in both places.

Reply:
I can backport this when it goes in as needed